docs/tools.rst

```diff
@@ -71,6 +71,7 @@ Some differences between bgfx's shaderc flavor of GLSL and vanilla GLSL:
 -  ``bool/int`` uniforms are not allowed; all uniforms must be ``float``.
 -  Attributes and varyings can only be accessed from ``main()``.
 -  ``SAMPLER2D/3D/CUBE/etc.`` macros replace the ``sampler2D/3D/Cube/etc.`` tokens.
+- Sampler names should not end with ``Texture``, ``Sampler``, or ``SamplerComparison``.
```

Comment:
There is no issue with actually using that in name...

Reply:
You should read the message I linked to, which is a case of an issue with using that in the name: https://discord.com/channels/712512073522872352/712512073522872355/1132718863063195739

Reply:
I see, there is actually bug in code, it's using searching for "Texture" or "Sampler" instead checking does name have suffix. I'm going to fix that instead.